source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/AdaptiveFileBrowserSource.mm

```diff
@@ -45,7 +46,13 @@ - (instancetype)initWithFrame:(CGRect)frame
         if (hostconfigParseResult.isValid) {
             _hostConfig = hostconfigParseResult.config;
         }
-
+#if RELEASE
+        // the most of cards below v1.3 don't have updated accssibility features, but these cards do serve well as visualization test during bug bash
+        // so instead of removing them, excluded these cards when built for release.
+        _restrictedPaths = [NSSet setWithObjects:@"v1.0", @"v1.1", @"HostConfig", @"Templates", @"Elements", @"Tests", nil];
+#else
+        _restrictedPaths = [NSSet setWithObjects:@"HostConfig", @"Templates", nil];
```

Comment:
What happen if remove #if RELEAE #else and use `_restrictedPaths = [NSSet setWithObjects:@"v1.0", @"v1.1", @"HostConfig", @"Templates", @"Elements", @"Tests", nil];`

Reply:
then devs can't use cards in v1.0, v1.1, and all of the elements and test folders. That is not desirable since these cases provide values in debugging visual issues.